src/aap_eda/services/activation/db_log_handler.py

```diff
@@ -47,6 +47,7 @@ def write(
         lines: Union[list[str], str],
         flush: bool = False,
         timestamp: bool = True,
+        log_timestamp: int = 0,
```

Comment:
Is correct that default value? I think we expect always a value.

Reply:
0 should be ok. It only happens in the beginning when the lines are added by us.